systemvm/debian/opt/cloud/bin/setup/cloud-early-config

```diff
@@ -93,6 +93,7 @@ start() {
   sync
   /opt/cloud/bin/setup/bootstrap.sh
 
+  modprobe nf_nat_pptp
```

Comment:
why must this one be explicitely be loaded while the other nat and conntrack modules are 'just' written to /etc/modules?

Reply:
@DaanHoogland it requires rebuild of systemvm template I guess ?

Reply:
yes, the other file, tools/appliance/systemvmtemplate/scripts/configure_conntrack.sh
So if we have a systemvm rebuild we can remove this again? It will be part of the systemvm.iso and the other bit should allready have loaded it.